tests/unit/session-stores/shared/store-behavior.js

```diff
@@ -1,4 +1,5 @@
-/* jshint expr:true */
+/* eslint-disable mocha/no-top-level-hooks */
```

Comment:
Can this be disabled only for the one `beforeEach` in line 15 so we don't switch it off for the whole file?